challenge_storing_groceries/src/challenge_storing_groceries/manipulate_machine.py

```diff
@@ -82,7 +83,7 @@ def __init__(self, robot, grab_designator=None):
         smach.StateMachine.__init__(self, outcomes=["succeeded", "failed"])
 
         # Create designators
-        self.empty_arm_designator = ds.UnoccupiedArmDesignator(robot, {}, name="empty_arm_designator")
+        self.empty_arm_designator = ds.OccupiedArmDesignator(robot, {}, name="empty_arm_designator")
```

Comment:
Why change this from Unoccupied to Occupied?

Reply:
Because I did an oopsie